src/controllers/bulkUpload.ts

```diff
@@ -42,16 +56,16 @@ export const fileUpload = async (ctx) => {
   let response;
   try {
     response = await destFileUploadHandler.processFileData(ctx.request.body);
-  } catch (error: any) {
+  } catch (error: CatchErr) {
     response = {
-      statusCode: error?.response?.status || error?.status || 400,
-      error: error.message || 'Error occurred while processing payload.',
+      statusCode: error.response ? error.response.status : 400,
```

Comment:
`error.status` is not required at all ? Did we confirm about this ?

Reply:
reverted